workflows/scRNAseq/standard-scanpy/Standard-scRNA-seq-with-Scanpy.ga

```diff
@@ -0,0 +1,3096 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Standard scRNA-seq workflow with Scanpy and Anndata. Based on the 3k PBMC clustering tutorial from Scanpy. Important workflow parameters can be read from a tabular file.",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Person",
+            "identifier": "0000-0002-5192-126X",
+            "name": "Pavankumar Videm"
+        },
+        {
+            "class": "Person",
+            "identifier": "0000-0002-2799-424X",
+            "name": "Hans-Rudolf Hotz"
+        },
+        {
+            "class": "Person",
+            "identifier": "0000-0002-4181-2676",
+            "name": "Mehmet Tekman"
+        },
+        {
+            "class": "Person",
+            "identifier": "0000-0001-9852-1987",
+            "name": "B\u00e9r\u00e9nice Batut"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "release": "0.1",
+    "name": "Standard scRNA-seq with Scanpy",
```

Comment:
```suggestion
    "name": "scRNA-seq with Scanpy",
```

Reply:
I will rename it to "Preprocessing and CLustering with Scanpy".